client/src/components/screens/Patient/components/FamilyTab/index.tsx

```diff
@@ -67,8 +66,8 @@ const FamilyTab = (): React.ReactElement => {
       ) : (
         <FamilyTable
           addParentMenu={menu}
+          allowActions={isPatientProband}
```

Comment:
**renaming**: `allowActions` means "okay, we show the actions column as a whole. However, it does not mean that every action will be display for every member."